src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    public Square(String color, String name) {
+        setColor(color);
+        setName(name);
+
+    }
+
+    @Override
+    public double getArea() {
+        return super.getArea();
+    }
+
+    @Override
+    public void draw() {
+        super.draw();

```

Comment:
The draw() method should be overridden to provide the specific drawing logic for a Square. Calling super.draw() is not sufficient as it does not define the specific drawing behavior for the Square class.